src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts

```diff
@@ -187,6 +199,22 @@ export class DotEditContentHtmlService {
         return this.getEditPageIframe().contentWindow.getDotNgModel();
     }
 
+    private handleIframeClicks(target: HTMLElement) {
+        if (target.dataset.dotObject === 'edit-content') {
```

Comment:
this 'edit-content' shouls be a const somewhere

Reply:
When we do the refactor we talked about it on how to handle all the clicks in the iframe.window we do it, right now is just one action that is being handle it like this.